MAINTAINERS.md

```diff
@@ -1,14 +1,17 @@
-## Maintainer
+## Maintainers
 
-They are approvers who have shown good technical judgement in feature design/development in the past. Has overall knowledge of the project and features in the project. These role is for project managers who are expected to manage the repository without access to sensitive or destructive actions.
+Maintainers are approvers who have shown good technical judgement in guiding feature design & development, have displayed overall knowledge of the project and features in the project, and are nuturing and receptive to everyone in the community.
 
-| Maintainer  | GitHub ID                                   | Affiliation               |
-| ----------- | ------------------------------------------- | ------------------------- |
-| JimBugwadia | [@JimBugwadia](https://github.com/JimBugwadia) | Nirmata Inc.              |
-| Realshuting | [@realshuting](https://github.com/realshuting) | Nirmata Inc.              |
-| Chip zoller | [@chipzoller](https://github.com/chipzoller)  | dell                      |
-| Marcel Mue  | [@MarcelMue](https://github.com/MarcelMue)   | Giant Swarm GmbH          |
-| Treydock    | [@treydock](https://github.com/treydock)    | Ohio Supercomputer Center |
+| Maintainer            | GitHub ID                                              | Affiliation               |
+| --------------------- | ------------------------------------------------------ | ------------------------- |
+| Jim Bugwadia          | [@JimBugwadia](https://github.com/JimBugwadia)         | Nirmata                   |
+| Shuting Zhao          | [@realshuting](https://github.com/realshuting)         | Nirmata                   |
+| Chip Zoller           | [@chipzoller](https://github.com/chipzoller)           | Dell Technologies         |
+| Marcel Müller         | [@MarcelMue](https://github.com/MarcelMue)             | Giant Swarm GmbH          |
+| Trey Dockendorf       | [@treydock](https://github.com/treydock)               | Ohio Supercomputer Center |
+| Vyankatesh Kudtarkar  | [@vyankyGH](https://github.com/vyankyGH)               | Nirmata                   |
+| Prateek Pandey        | [@prateekpandey14](https://github.com/prateekpandey14) | Nirmata                   |
+| Sambhav Kothari       | [@samj1912](https://github.com/samj1912)               | Bloomberg                 |
 
 
-**Note**: Please refer to the [Project Governance](https://kyverno.io/community/project-governance/) for more information on the responsibilities and privileges of a maintainer in Kyverno.
+**Note**: Please refer to the [Project Governance](https://main.kyverno.io/community//project-governance/) for more information on the responsibilities and privileges of a maintainer in Kyverno.
```

Comment:
Two forward slashes in this link now.

Reply:
Missing octothorp (anchor)

Reply:
Rewriting my comment https://github.com/kyverno/kyverno/pull/3155#discussion_r797819498 here from the previously dismissed stale review:

>Oh, I see. But the link provided in this file is `https://kyverno.io/community/project-governance/` which is missing the `#` symbol before project-governance and has an additional `/` at the end, which will cause it to not redirect the user's screen to the `Project Governance` section of the webpage, even though the webpage itself would be working properly.